--- pinfo-0.6.10/src/filehandling_functions.c.bero	2012-04-03 20:39:59.976157425 +0200
+++ pinfo-0.6.10/src/filehandling_functions.c	2012-04-03 20:42:59.219158315 +0200
@@ -41,14 +41,18 @@ char * basename(char *filename);
  * you dump the file through `gunzip -d -c', etc.                              *
  ******************************************************************************/
 
-#define SuffixesNumber 4
+#define SuffixesNumber 8
 
 Suffixes suffixes[SuffixesNumber] =
 {
 	{"", 		"cat"},
 	{".gz",		"gzip -d -q -c"},
 	{".Z",		"gzip -d -q -c"},
-	{".bz2",	"bzip2 -d -c"}
+	{".bz2",	"bzip2 -d -c"},
+	{".lzma",	"lzma -d -q -c"},
+	{".xz",		"xz -d -c"},
+	{".lz",		"lzip -d -c"},
+	{".zst",	"zstd -d -c"}
 };
 
 /*****************************************************************************/
--- pinfo-0.6.9/doc/pinfo.1.in.lzma_support	2007-09-24 20:47:59.000000000 +0200
+++ pinfo-0.6.9/doc/pinfo.1.in	2007-09-24 20:48:21.000000000 +0200
@@ -19,7 +19,8 @@ The program will then (by default) searc
 and
 .IR "/opt/info".
 The search path can be adjusted by INFOPATH environment variable or in the configuration
-file. Pinfo will also automatically add the suffix '-info', '-info.Z', '-info.gz', or '-info.bz2'.
+file. Pinfo will also automatically add the suffix '-info', '-info.Z', '-info.gz', '-info.bz2',
+'-info.lzip', '-info.xz', '-info.lzma' or '-info.zst'.
 At present other suffixes are not recognized, but you can easily add them to
 the function 
 .I openinfo() 
--- pinfo-0.6.9/doc/pinfo.texi.lzma_support	2007-09-24 20:44:16.000000000 +0200
+++ pinfo-0.6.9/doc/pinfo.texi	2007-09-24 20:49:35.000000000 +0200
@@ -108,8 +108,9 @@ will then (by default) search for it in 
 and @file{/opt/info}. Other searchpath may be
 specified in configfile or via INFOPATH environmental variable.
 Pinfo will also automaticaly add the suffix
-@samp{-info}, @samp{-info.Z}, @samp{-info.gz}, or @samp{-info.bz2}. At
-present other suffixes are not recognized, but you can easily add them
+@samp{-info}, @samp{-info.Z}, @samp{-info.gz}, @samp{-info.bz2}, @samp{-info.lzma},
+@samp{-info.lzip}, @samp{-info.xz} or @samp{-info.zst}.
+At present other suffixes are not recognized, but you can easily add them
 to the function @code{openinfo()} in @file{filehandling_functions.c}.
 
 When the search for info pages fails, man is called with the @var{info_page}
