--- pinfo-0.6.10/src/filehandling_functions.c.bero	2012-04-03 20:39:59.976157425 +0200
+++ pinfo-0.6.10/src/filehandling_functions.c	2012-04-03 20:42:59.219158315 +0200
@@ -41,14 +41,17 @@ char * basename(char *filename);
  * you dump the file through `gunzip -d -c', etc.                              *
  ******************************************************************************/
 
-#define SuffixesNumber 4
+#define SuffixesNumber 7
 
 Suffixes suffixes[SuffixesNumber] =
 {
 	{"", 		"cat"},
 	{".gz",		"gzip -d -q -c"},
 	{".Z",		"gzip -d -q -c"},
-	{".bz2",	"bzip2 -d -c"}
+	{".bz2",	"bzip2 -d -c"},
+	{".lzma",	"lzma -d -q -c"},
+	{".xz",		"xz -d -c"},
+	{".lz",		"lzip -d -c"}
 };
 
 /*****************************************************************************/
